src/app/design/frontend/base/default/template/checkout/omise/paynow.phtml

```diff
@@ -0,0 +1,82 @@
+<div class="page-title">
```

Comment:
Given the similarity between this template and Tesco, it would probably make sense to base each one on a common template and fill in just the bits that are different

Reply:
I notice this was already done for the CSS

Reply:
Leaving template things (phtml, email) to be done in future refactoring PR as barcode and barcode reference section is quite tricky to implement as one being img link, other being html code iteself.